mybatis/deployment/src/main/java/io/quarkiverse/mybatis/deployment/MyBatisProcessor.java

```diff
@@ -207,7 +206,6 @@ void generateSqlSessionFactory(MyBatisRuntimeConfig myBatisRuntimeConfig,
                                     configurationCustomizers,
                                     sqlSessionFactoryBuilderBuildItem.getBuilder(),
                                     myBatisRuntimeConfig,
-                                    dataSourceConfig,
```

Comment:
You actually cannot pass partial config objects to recorder. So here I will extract `MyBatisDataSourceRuntimeConfig` in the recorder instead.

Reply:
OK, it good to me. Thanks for your explanation!